backend/cn/lib/builtins.ml

```diff
@@ -5,14 +5,13 @@ open Effectful.Make (Resultat)
 
 open TypeErrors
 open IndexTerms
+module CF = Cerb_frontend
 
-(* builtin function symbols *)
-let mk_arg0 mk args loc =
-  match args with
-  | [] -> return (mk loc)
-  | _ :: _ as xs ->
-    fail { loc; msg = Number_arguments { has = List.length xs; expect = 0 } }
+type builtin_fn = LogicalFunctions.definition
 
+let builtin_location = Cerb_location.other "<builtin>"
```

Comment:
This can just be called `loc` for short - the context will make it clear, and it will allow for record punning in records.